src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
as we must return something from Colors enum better will be:

```suggestion
    public static Colors getRandomColor() {
        int index = new Random().nextInt(Colors.values().length);
        return Colors.values()[index];
```
sure, we can skip line with finding index and paste all code instead "[index]", but don't forger about code readability!

also, in my opinion, better to move this method with random picking color to Colors enum class.
sure, best practice to leave enum class as simple enum only with constants, but this method very related to Colors